src/doc/CreateHighlightDialog.js

```diff
@@ -84,15 +85,16 @@ class CreateHighlightDialog extends EventEmitter {
         }
 
         // Select page of first node selected
-        this.pageInfo = getPageInfo(selection.anchorNode);
+        this.selection = selection;
+        this.pageInfo = getPageInfo(this.selection.anchorNode);
         if (!this.pageInfo.pageEl) {
             return;
         }
 
         if (this.isMobile) {
             this.position = { x: 0, y: 0 };
         } else {
-            this.setPosition(selection);
+            this.setPosition();
```

Comment:
Common methods like `setPosition()`, `setDimensions()`, etc. are not widely acceptable if they do not/can not take in parameters. I would suggest renaming the function to `updatePosition()`